src/misc.c

```diff
@@ -581,21 +580,55 @@ char *human_prefix(uint64_t num)
 	return out;
 }
 
-char *human_prefix_small(double num)
+char *human_speed(uint64_t speed)
```

Comment:
I'd rename `human_prefix` to e.g. `human_prefix_binary`, and then `human_speed` to `human_prefix_decimal` (or `human_prefix_large`?) and omit the " c/s" from it (to be added by the callers). I guess we might need it in places where the speed isn't in c/s, but is e.g. in p/s. (We can't simply use this function for our status line, though, at least because the memory leak would unacceptable there. So let's not do that just yet.)